README.md

```diff
@@ -1 +1,3 @@
-# Gravity-Forms
```

Comment:
This needs to be converted to readme.txt as per the wordpress standards. See [this](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/) and [the validator](https://wordpress.org/plugins/developers/readme-validator/)